src/main/java/com/fuzs/aquaacrobatics/core/mixin/EntityPlayerMixin.java

```diff
@@ -309,7 +316,11 @@ public boolean isResizingAllowed() {
     }
 
     protected float getEyeHeight(Pose poseIn, EntitySize sizeIn) {
-
+        if (WitcheryResurrectedIntegration.isWolfTransformation(this.getPlayer())) {
```

Comment:
Ideally the call to `IntegrationManager.isWitcheryResurrectedEnabled()` should be moved out here to avoid potential undesired classloading.